### ISSUE TITLE
language support and rtl version

### ISSUE THREAD

**Issue by MetaiR** [elmsln/elmsln#1950]
Feature request:
hello,
does it support languages like Persian? (or is translating possible easily?)

and does it support rtl theme? (or is it possible and easy to write a theme for it?)

**Comment by btopro** [comment]
I'm nots sure what Persian is but elmsln's core functionality is built on drupal which is excellent for language compatibility / flexibility. We do encapsulate all strings with the t() function which allows for translations but I don't know much about doing the RTL / drupal specific language translation file integration, I just know it is possible and we would definitely add support if guided in ways to best do so.

**Comment by MetaiR** [comment]
really thanks
LMS and online tutorials are going to be very popular in my country (Iran) these days.
btw:
Persian  = Farsi -> Iranian language

**Comment by btopro** [comment]
https://www.drupal.org/project/fa gotcha, thanks for clarifying, looks like there is some work towards .po translation files in drupal. If you run into any issues associated with doing this let me know, this would be very exciting to see added / help with spreading elms beyond US/UK involvement. Part of the reason for picking Drupal was the translation engine in order to open the possibility for things like this.

**Comment by MetaiR** [comment]
does it happen yet?

**Comment by btopro** [comment]
There is development level support for it but I do not have any plans at this time of adding this without someone sponsoring and doing it. I haven't implemented translated Drupal sites; just know that it should be possible to translate top to bottom

**Comment by btopro** [comment]
these is core support for this tho translations would be needed; closing for now
